.travis.yml

```diff
@@ -161,7 +155,7 @@ before_script:
   - if [ "$DB" = 'pgsql' ]; then psql -c 'CREATE DATABASE bedita_test;' -U postgres; fi
   - if [ "$DB" = 'pgsql' ] && [ "$DB_POSTGIS" = '1' ]; then psql -c 'CREATE EXTENSION postgis;' -U postgres; fi
 
-script: vendor/bin/phpunit --coverage-clover=clover.xml # Run PHPUnit.
```

Comment:
The clover.xml is used for Codecov https://codecov.io in `after_success` there is the script to upload it.
Could it be created just in one case?